### DISCUSSION TITLE
Need help, Main process does not allow to execute a node/express webserver

### DISCUSSION THREAD

**Discussion by otekdo** [alex8088/electron-vite#106] [Q&A] [answered]
Hello guys, previously i asked if it was possible to setup an express server, but now that i'm trying, everytime. 
I try to launch a node process on ``main`` process even before creating the window, i'm always getting the following issue:
![image](https://user-images.githubusercontent.com/65221868/218279784-981b8c1d-7705-48dd-8c2a-296d0a29e041.png)

**NOTE:** If i am using the `electron forge` boilerplate it works, but with `electron-vite`, it does not work
I'm trying to use `electron-vite` because development productivity is way better than `electron-forge`

This is how the directory is set, is just an initial React+Typescript template i created with `yarn create @quick-start/electron`
![image](https://user-images.githubusercontent.com/65221868/218280071-6600bc77-11b3-4643-8841-7841c7b617b3.png)

I only have a minimal express web app on the background:
**server/index.ts**
```typescript
const express = require('express')

const app = express()
const port = 5000

app.get('/', (req, res) => {
  res.send('Hello World!')
})

app.listen(port, () => {
  console.log(`Example app listening on port ${port}`)
})
```

**main/index.ts**
```typescript
import { app, shell, BrowserWindow } from 'electron'
import { join } from 'path'
import { electronApp, optimizer, is } from '@electron-toolkit/utils'
import icon from '../../resources/icon.png?asset'

require('./server/index')

// ... REMAINING CODE FROM BOILERPLATE
```

Weirdly it works with `electron-forge + webpack` boilerplate, but it does not work with this toolset
Do i need to take an additional step in order to launch an `express` webserver with `electron-vite`?

**Comment by alex8088** [comment, marked as answer]
You need to use `es` syntax.

``` js
import express from 'express'
//…
```

and

```js
import './server/index'
```

**Reply by otekdo** [reply]
Thanks, it worked!

**Comment by dmZhan** [comment]
Amazing!
